csharp/ql/src/semmle/code/csharp/controlflow/internal/Splitting.qll

```diff
@@ -445,7 +445,7 @@ module AssertionSplitting {
 
     override predicate hasEntry(ControlFlowElement pred, ControlFlowElement succ, Completion c) {
       exists(AssertMethod m |
-        pred = last(a.getExpr(), c) and
+        pred = last(a.getAnExpr(), c) and
```

Comment:
The CFG splitting logic will not work for assertions with multiple assertion arguments; I will address in a follow-up PR.